### PR TITLE
fixed numerous styling issues and removed a lot of placeholder content

### PR DESCRIPTION
**Pull request by jake-rasmussen**

### CODE REVIEW

**Comment by jake-rasmussen** [comment]
Neatened up some spacing issues and fixed hydration issues

File: src/pages/index.tsx [hunk 1]
```diff
@@ -45,17 +45,16 @@ const Home: NextPageWithLayout = () => {
 
   const slides = images.map((image, index) => {
     return (
-      <>
+      <React.Fragment key={index}>
```
Comment (d-drozdov) [inline src/pages/index.tsx:48]: Do we need to import React bc of this ?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import { useRef } from "react";
+import React, { useRef } from "react";
 import type { NextPageWithLayout } from "./_app";
 import { Carousel } from "@mantine/carousel";
 import Autoplay from "embla-carousel-autoplay";
 import sample from "../../assets/images/sample.png";
 import banner from "../../assets/images/whizkidsbanner.png";
 import Image from "next/image";
-import MainLayout from "../components/layouts/MainLayout";
+import MainLayout from "../components/layouts/mainLayout";
```
Comment (d-drozdov) [inline src/pages/index.tsx:9]: Should be "../components/layouts/MainLayout" I believe

File: src/components/layouts/mainLayout.tsx [hunk 3]
```diff
@@ -4,11 +4,11 @@ import NavBar from "../navbar";
 
 const MainLayout: LayoutProps = ({ children }) => {
   return (
-    <>
+    <div className="overflow-y-scroll">
```
Comment (d-drozdov) [inline src/components/layouts/mainLayout.tsx:7]: Not sure what this is for maybe you can tell me later

**Review by d-drozdov — CHANGES_REQUESTED** [review]
Just take a look at some of the comments

File: src/pages/index.tsx (hunk 1 above)
Comment (jake-rasmussen) [reply src/pages/index.tsx]: Yes but it's imported already

**Review by d-drozdov — APPROVED** [review]
LGTM

**Pull request merged**